<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_league_dialog</class>
 <widget class="QDialog" name="edit_league_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit League</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="teams_list_widget">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="import_csv_button">
         <property name="text">
          <string>Import Teams</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="export_csv_button">
         <property name="text">
          <string>Export Teams</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add_team_button">
       <property name="text">
        <string>Add Team</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit_team_button">
       <property name="text">
        <string>Edit Team</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_team_button">
       <property name="text">
        <string>Delete Team</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
